net-misc/gns3-server/gns3-server-2.1.16.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{5,6} )
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="GNS3 server to asynchronously manage emulators"
+HOMEPAGE="http://www.gns3.net/"
+SRC_URI="https://github.com/GNS3/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+RESTRICT="test"
```

Comment:
This needs explanation in a form of comment. A good explanation.

Reply:
inherited from old ebuild in tree, I'll drop this, since it seems pretty old